Documentation/operations/system_requirements.rst

```diff
@@ -194,6 +194,27 @@ The following kernel configuration options are required for proper operation:
         CONFIG_GENEVE=y
         CONFIG_FIB_RULES=y
 
+
+⚠️ Note on CONFIG_FIB_RULES Activation
+
+On some embedded or custom Linux systems—especially when cross-compiling for 
+ARM—enabling ``CONFIG_FIB_RULES=y`` directly in the kernel ``.config`` may not be sufficient, 
+as it depends on other routing-related kernel options to be enabled.
+
+If ``CONFIG_FIB_RULES`` is missing from the final compiled kernel (e.g., not visible via cat /boot/config-*), 
```

Comment:
If not escaped, the asterisk is interpreted as a marker for emphasis (slanted text).

```suggestion
If ``CONFIG_FIB_RULES`` is missing from the final compiled kernel (e.g., not visible via cat /boot/config-\*), 
```